lib/node_modules/@stdlib/stats/base/ndarray/dmeanstdev/lib/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Compute the mean and standard deviation of a one-dimensional double-precision floating-point ndarray.
+*
+* @module @stdlib/stats/base/ndarray/dmeanstdev
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+* var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+* var ndarray = require( '@stdlib/ndarray/base/ctor' );
+* var dmeanstdev = require( '@stdlib/stats/base/ndarray/dmeanstdev' );
+*
+* var opts = {
+*     'dtype': 'float64'
+* };
+*
+* var xbuf = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
+* var x = new ndarray( opts.dtype, xbuf, [ 4 ], [ 2 ], 1, 'row-major' );
+* var out = new ndarray( opts.dtype, new Float64Array( 2 ), [ 2 ], [ 1 ], 0, 'row-major' );
+* var correction = scalar2ndarray( 1.0, opts );
+*
+* var v = dmeanstdev( [ x, out, correction ] );
+* // returns <ndarray>
```

Comment:
You should actually display the expected results (i.e., the mean and stdev) using `ndarray2array`.